Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Pull base image
+FROM python:3.8.3-alpine
+
+# Set work directory
+WORKDIR /usr/src/app
+
+# Set environment variables
+# Prevents Python from writing pyc files
+ENV PYTHONDONTWRITEBYTECODE 1
+# Prevents Python from buffering stdout and stder
+ENV PYTHONUNBUFFERED 1
+
+# Install Pillow dependencies (fuck_you!)
+RUN apk add --no-cache jpeg-dev zlib-dev
+RUN apk add --no-cache --virtual .build-deps build-base linux-headers
```

Comment:
This is sad :(. If we were able to avoid the system libraries we could do a multi-stage build with very small footprint, just a 50 Mb or so of Docker image